### PR TITLE
chapter_8: Log in

### PR DESCRIPTION
**Pull request by ngyihkai**
Ready

### CODE REVIEW

**Comment by ngyihkai** [comment]
Ready

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/sessions.scss]: delete this file because it not use

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,17 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove it
because you defined at line 13

File: config/routes.rb [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
+  get 'sessions/create'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove it

File: config/routes.rb [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
+  get 'sessions/destroy'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove it

File: test/controllers/sessions_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+require "test_helper"
+
+class SessionsControllerTest < ActionDispatch::IntegrationTest
```
Comment (ngocvt-0484) [inline test/controllers/sessions_controller_test.rb]: dont need do test. please delete this file

**Comment by ngyihkai** [comment]
Ready

File: app/views/sessions/create.html.erb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#create</h1>
```
Comment (ghost) [inline app/views/sessions/create.html.erb]: please delete this file if you don't use it

File: app/views/sessions/destroy.html.erb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#destroy</h1>
```
Comment (ghost) [inline app/views/sessions/destroy.html.erb]: please delete this file if you don't use it

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ngyihkai** [comment]
Ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**